src/dotnet/HQ.Server/Services/ProjectServiceV1.cs

```diff
@@ -237,6 +237,13 @@ public ProjectServiceV1(ChargeCodeServiceV1 chargeCodeServiceV1, HQDbContext con
         {
             records = records.Where(t => t.Status == request.ProjectStatus);
         }
+        if (request.CurrentOnly.HasValue)
+        {
+            if (request.CurrentOnly.Value)
+            {
+                records = records.Where(t => t.Status == ProjectStatus.InProduction || t.Status == ProjectStatus.Ongoing);
+            }
+        }
```

Comment:
This one can be simplified to 
`if (request.CurrentOnly == true)
{
    records = records.Where(t => t.Status == ProjectStatus.InProduction || t.Status == ProjectStatus.Ongoing);
}`